webapp/utils/text_formatting.jsx

```diff
@@ -162,12 +162,23 @@ export function autolinkAtMentions(text, tokens, usernameMap) {
         return (Constants.SPECIAL_MENTIONS.indexOf(u) !== -1 || Boolean(usernameMap[u]));
     }
 
+    function isSpecialMention(username) {
```

Comment:
I tried but I couldn't make it pass on several test cases. Not sure how could I change `mentionExists` function considering special mentions to be highlighted but non-clickable. `isSpecialMention` is now deleted but just use this `Constants.SPECIAL_MENTIONS.indexOf(username) >= 0` as a replacement. Happy to know how you'd like to change it :)